pom.xml

```diff
@@ -95,12 +95,6 @@
 			<version>RELEASE</version>
 			<scope>compile</scope>
 		</dependency>
-		<dependency>
-			<groupId>org.jetbrains</groupId>
-			<artifactId>annotations</artifactId>
-			<version>RELEASE</version>
-			<scope>compile</scope>
-		</dependency>
```

Comment:
If this change gets merged wouldn't other face problems?

Reply:
the same dependency was added twice. So I removed one